Description: Use a common naming scheme for archives created by one of the
 database backup methods.
Author: Georgios M. Zarkadas <gz@member.fsf.org>
Forwarded: https://github.com/sukria/Backup-Manager/pull/16
Last-Update: 2011-12-25

--- a/lib/backup-methods.sh
+++ b/lib/backup-methods.sh
@@ -939,7 +939,7 @@
             file_to_create="$BM_REPOSITORY_ROOT/${BM_ARCHIVE_PREFIX}-all-pgsql-databases.$TODAY.sql"
             command="${pgdump}all $opt $BM_PGSQL_EXTRA_OPTIONS"
         else
-            file_to_create="$BM_REPOSITORY_ROOT/${BM_ARCHIVE_PREFIX}-${database}.$TODAY.sql"
+            file_to_create="$BM_REPOSITORY_ROOT/${BM_ARCHIVE_PREFIX}-pgsql-${database}.$TODAY.sql"
             command="$pgdump $opt $database $BM_PGSQL_EXTRA_OPTIONS"
         fi
         __create_file_with_meta_command
@@ -997,7 +997,7 @@
             file_to_create="$BM_REPOSITORY_ROOT/${BM_ARCHIVE_PREFIX}-all-mysql-databases.$TODAY.sql"
             command="$base_command --all-databases"
         else
-            file_to_create="$BM_REPOSITORY_ROOT/${BM_ARCHIVE_PREFIX}-${database}.$TODAY.sql"
+            file_to_create="$BM_REPOSITORY_ROOT/${BM_ARCHIVE_PREFIX}-mysql-${database}.$TODAY.sql"
             command="$base_command $database"
         fi
         __create_file_with_meta_command
